core/modules/system/css/system.theme.css

```diff
@@ -496,6 +496,18 @@ tr:hover del.diffchange {
   margin-bottom: 50px;
 }
 
+/**
+ * External link indicators.
+ */
+span.external {
+  display: inline-block;
+  margin-left: .33em;
```

Comment:
...we need an RTL rule for this:

<img width="1226" alt="Screen Shot 2019-09-16 at 8 05 24 am" src="https://user-images.githubusercontent.com/2423362/64928252-d5f78180-d858-11e9-9c89-5dd1d091ba76.png">